Test.elm

```diff
@@ -1,11 +1,7 @@
-module Test exposing (..)
```

Comment:
It was causing a circular dependency, since I needed

'import Test' (its no longer ElmTest). 

I thought about changing it to 

'module FlatMatrixTests' but I figured since it was not imported anywhere, why not remove it. Is there a better solution?